src/services/GroupMeService.ts

```diff
@@ -19,11 +19,12 @@ export class GroupMeService implements MessagingService {
    * Sends a message to GroupMe.
    * @param text
    */
-  async sendMessage(text: string): Promise<Either<Error, number>> {
+  async sendMessage(text: string, attachments: GroupMeAttachment[] = []): Promise<Either<Error, number>> {
```

Comment:
Any reason for default param over possibly undefined?

Reply:
https://github.com/JosephMart/loki-gm/pull/14/files#diff-0135c17db556ed19014403cc5c86b0a1R27